src/test/lib/server/server_test_runner.cpp

```diff
@@ -124,13 +124,13 @@ TEST_F(ServerTestRunner, TestInvalidCopyImport) {
   pqxx::nontransaction transaction{connection};
 
   // Ill-formed
-  EXPECT_THROW(transaction.exec("COPY another_table FROM;"), pqxx::sql_error);
+  EXPECT_THROW(transaction.exec("COPY another_table FROM;"), pqxx::broken_connection);
```

Comment:
Wouldn't `pqxx::sql_error` be more accurate?
Same for the changes below.

Reply:
> Wouldn't `pqxx::sql_error` be more accurate? Same for the changes below.

Yes, but with the libpqxx version used in this PR, the `exec` function of that third-party library no longer throws the `sql_error`. It was changed to the `broken_connection` (see the links TODO (4) of this PR's description).